plugins/techdocs/src/alpha.tsx

```diff
@@ -13,13 +13,37 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { default as React } from 'react';
 
-import React from 'react';
 import {
+  configApiRef,
+  createApiFactory,
+  createRouteRef,
+  discoveryApiRef,
+  fetchApiRef,
+  identityApiRef,
+} from '@backstage/core-plugin-api';
+import {
+  createApiExtension,
+  createPageExtension,
   createPlugin,
   createSchemaFromZod,
 } from '@backstage/frontend-plugin-api';
 import { createSearchResultListItemExtension } from '@backstage/plugin-search-react/alpha';
+import {
+  techdocsApiRef,
+  techdocsStorageApiRef,
+} from '@backstage/plugin-techdocs-react';
+import { TechDocsClient, TechDocsStorageClient } from './client';
+
+const rootRouteRef = createRouteRef({
+  id: 'plugin.techdocs.indexPage',
+});
+
+const rootDocsRouteRef = createRouteRef({
+  id: 'plugin.techdocs.readerPage',
+  params: ['namespace', 'kind', 'name'],
+});
```

Comment:
Suggestion (non-blocking): 
Use existing route references.
https://github.com/backstage/backstage/blob/6feb4c7d66da6ad34754aa4064d09163db4451d5/plugins/techdocs/src/routes.ts#L19-L26

Reply:
ok - let's keep it until we solve the id issue :)